cookiecutter.json

```
@@ -1,15 +1,29 @@
 {
   "full_name": "First Middle Last",
   "email": "email@example.com",
-  "github_username": "<github username>",
   "autora_contribution_type": ["theorist", "experimentalist", "experiment_runner", "synthetic_data", "custom"],
+  "custom_autora_contribution_type": "{% if cookiecutter.autora_contribution_type == 'custom' -%}<custom_autora_contribution_type>{% else -%}N/A - Press Enter to Skip{% endif -%}",
+  "__autora_contribution_type": "{% if cookiecutter.autora_contribution_type == 'custom' -%}{{ cookiecutter.custom_autora_contribution_type }}{% else -%}{{ cookiecutter.autora_contribution_type }}{% endif -%}",  
+  "experimentalist_type": [
+    "{% if cookiecutter.autora_contribution_type == 'experimentalist' -%}sampler{% else -%}N/A - Press Enter to Skip{% endif -%}", 
+    "{% if cookiecutter.autora_contribution_type == 'experimentalist' -%}pooler{% else -%}N/A - Press Enter to Skip{% endif -%}", 
+    "{% if cookiecutter.autora_contribution_type == 'experimentalist' -%}custom{% else -%}N/A - Press Enter to Skip{% endif -%}"],
```

If this whole inner list was wrapped in an if/else construct would it simplify the rendered output when it it is NA? 
```suggestion
    "{% if cookiecutter.autora_contribution_type == 'experimentalist' %} ['sampler', 'pooler', 'custom'] {% else -%}'N/A - Press Enter to Skip'{% endif -%}"]
```
Would something like this work?

---

No, this doesn't work. It would be a string instead of a list. So the default value would be the whole list as a string (instead of the options)